### PR TITLE
Exports deid case name

### PR DESCRIPTION
**Pull request by biyeun**
Fixes issue with not being able to remove case name from an export
http://manage.dimagi.com/default.asp?181136#1009817

FML @dannyroberts @esoergel

### CODE REVIEW

File: corehq/ex-submodules/couchexport/models.py [hunk 1]
```diff
@@ -353,14 +353,23 @@ def displays_by_index(self):
...
+        special_transforms = [
+            USERNAME_TRANSFORM, CASENAME_TRANSFORM, OWNERNAME_TRANSFORM
+        ]
         for c in self.columns:
             if c.doc_type in display_column_types:
-                selected_cols.add(c.index)
-                yield c.to_config_format()
+                selected = True
+                if (isinstance(c.transform, SerializableFunction)
+                    and c.transform.dumps_simple() in special_transforms):
```
Comment (dimagimon) [inline corehq/ex-submodules/couchexport/models.py]: E129 visually indented line with same indent as next logical line

**Comment by dannyroberts** [comment]
:x:

```
======================================================================
FAIL: test_split_column (couchexport.tests.test_schema.SavedSchemaTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/couchexport/tests/test_schema.py", line 93, in test_split_column
    self._test_split_column([[1, 1, 1, 1, None]])
  File "corehq/ex-submodules/couchexport/tests/test_schema.py", line 81, in _test_split_column
    self.assertEqual(data['Export']['headers'], expected_headers)
AssertionError: Lists differ: [] != ['Split | a', 'Split | b', 'Sp...
Second list contains 5 additional elements.
First extra element 0:
Split | a
- []
+ ['Split | a', 'Split | b', 'Split | c', 'Split | d', 'Split | extra']
----------------------------------------------------------------------
```

File: corehq/ex-submodules/couchexport/models.py (hunk 1 above)
Comment (dimagimon) [inline corehq/ex-submodules/couchexport/models.py]: E129 visually indented line with same indent as next logical line

**Comment by biyeun** [comment]
@dannyroberts fixed tests (hopefully)

**Comment by dannyroberts** [comment]
:x: Fail looks real but I'm restarting it just in case

```
======================================================================
FAIL: test_include_duplicates (corehq.apps.reports.tests.test_form_export.FormExportTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/reports/tests/test_form_export.py", line 78, in test_include_duplicates
    self.assertEqual(data['Export']['headers'], ['Name'])
AssertionError: Lists differ: [] != ['Name']
Second list contains 1 additional elements.
First extra element 0:
Name
- []
+ ['Name']
----------------------------------------------------------------------
```

File: corehq/ex-submodules/couchexport/models.py (hunk 1 above)
Comment (dimagimon) [inline corehq/ex-submodules/couchexport/models.py]: E129 visually indented line with same indent as next logical line

**Comment by biyeun** [comment]
@dannyroberts finally, the tests have passed

**Comment by kaapstorm** [comment]
Reverting for <http://manage.dimagi.com/default.asp?182187>, <http://manage.dimagi.com/default.asp?182192> and <http://manage.dimagi.com/default.asp?182193>

**Pull request merged**